test/functional/test_site/expected/sub_site/index.html

```diff
@@ -20,8 +20,11 @@
     <link rel="icon" href="/test_site/favicon.png">
 </head>
 <body>
-<div id="app">
-    <div id="content-wrapper">
+    <div id="app">
+        
```

Comment:
Just wondering if the empty lines here are intended? 😅

Reply:
Most likely due to the line break between template variables within `Page.ejs`, might look into it (since it is kind of ugly :P)